src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.common.not-implemented.view :as not-implemented]
+            [quo2.components.settings.data-item.content.left-side :as left-side]
+            [quo2.components.settings.data-item.content.right-side :as right-side]))
+
+(def view-internal
+  (fn [{:keys [blur? card? icon-right? label description status size theme on-press title subtitle
```

Comment:
I can't quite see where you are getting `size` prop from for this configuration?
As I can see in figma there is no variant dependent on a size configuration

Reply:
<img width="201" alt="Screenshot 2023-08-23 at 15 58 26" src="https://github.com/status-im/status-mobile/assets/55688834/84f98426-3129-4d6f-8301-7affb6ff0762">
Size can be either `default` or `small`

Reply:
We'll im looking in the wrong place so 😅😆